floodviz/static/data/cities.json

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Just FYI, you can use single quotes around a python string and therefore preserve the double quotes.